src/OmniSharp.Roslyn.CSharp/Services/DocumentationConverter.cs

```diff
@@ -131,10 +132,40 @@ private static string GetCref(string cref)
 
         public static DocumentationComment GetStructuredDocumentation(string xmlDocumentation, string lineEnding)
         {
-            if (string.IsNullOrEmpty(xmlDocumentation))
-                return null;
             return DocumentationComment.From(xmlDocumentation, lineEnding);
         }
+
+        public static DocumentationComment GetStructuredDocumentation(ISymbol symbol, string lineEnding = "\n")
+        {
+            switch (symbol)
+            {
+                case IParameterSymbol parameter: return DocumentationComment.WithSummaryText(DocumentationHelper.GetParameterDocumentation(parameter, lineEnding));
```

Comment:
Typically the case code goes on the line after the case label.

Reply:
Wouldn't this be nicer if it was `New DocumentationComment(summaryText = DocumentationHelper.GetParameterer(...`?